dev_tools/tests/data/doc_test.yaml

```diff
@@ -0,0 +1,48 @@
+category: application
+doc: |
+  This is a definition for data to be archived by ICAT (http://www.icatproject.org/).
+  
+  .. text from the icatproject.org site
+  
+          the database (with supporting software) that provides an
+          interface to all ISIS experimental data and will provide
+          a mechanism to link all aspects of ISIS research from
+          proposal through to publication.
+type: group
+NXarchive(NXobject):
+  (NXentry)entry:
+    \@index:
+    title:
+    experiment_identifier(NX_CHAR):
+      doc: 
+       - |
+        "unique identifier for the experimenta
+          lkokö
+          sddfdfd"
+       - | 
+        "xref:
+          spec: <spec>
+          term: <term>
+          url: <url>"
```

Comment:
Wouldn't it be better to have this directly as a yaml sub-key, like so:
```yaml
doc:
   - xref:
     - spec: <spec>
     - term: <term>
     - url: <url>
```

I feel that's less ambiguous to write and easier to parse

Reply:
Though syntactically you are correct.
After having 2 days discussion we all decided (https://github.com/FAIRmat-NFDI/nexus_definitions/issues/87) to keep it as it is present.